app/_locales/en/messages.json

```diff
@@ -222,7 +222,8 @@
     "description": "The name of the application"
   },
   "approve": {
-    "message": "Approve spend limit"
+    "message": "Approve $1 spend limit",
```

Comment:
Hmm. Adding this substitution here but not to the other "approve" messages seems a bit.... questionable. It should be fine, but it's not great for users of other locales because now the token symbol will no longer be shown anywhere inline, and it's not great for translators because they don't have any way to know this changed.

Maybe we should delete the other "approve" messages? At least if it falls back to English, the token symbol is there. And frankly the app must be terribly difficult to use anyway for anyone who doesn't understand English well, as we're missing translations all over the place.